backend/src/services/cartRepository.js

```diff
@@ -0,0 +1,42 @@
+// cartRepository.js
+import db from './database'; // Import database connection 
```

Comment:
checkout the other repos, you can copy most of the logic from them to interact with the db (there is no `db` import)